src/Util.ts

```diff
@@ -354,6 +355,9 @@ class Util {
         try {
             const parsed = JSON.parse(html.split("var ytInitialData = ")[1].split(";</script>")[0]);
             data = parsed.contents.twoColumnWatchNextResults.results.results.contents;
+            const fd = openSync("./result.json", "w");
```

Comment:
why?

Reply:
My bad, forgot to cleanup. It's fixed now.